staging/src/k8s.io/pod-security-admission/policy/check_capabilities_restricted.go

```diff
@@ -136,6 +136,11 @@ func capabilitiesRestricted_1_22(podMetadata *metav1.ObjectMeta, podSpec *corev1
 }
 
 func capabilitiesRestricted_1_25(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) CheckResult {
+	// See KEP-127: https://github.com/kubernetes/enhancements/blob/308ba8d/keps/sig-node/127-user-namespaces/README.md?plain=1#L411-L447
+	if isUserNamespacesEnabledForPod(podSpec) {
+		return CheckResult{Allowed: true}
```

Comment:
CheckCapabilitiesRestricted overrides checkCapabilitiesBaselineID ... by short-circuiting here, this is relaxing past baseline so that *adding* *any* capability is allowed. Is that intentional, or was the intent to stop requiring them to *drop* all capabilities?

Reply:
Can you link to the part of the user namespaces design that describes how capabilities additions are mitigated by user namespaces. runAsUser and runAsNonRoot are much more clearly mitigated by the uid mapping.

Reply:
I linked the linux man pages and extended the comment. There are no further capability checks necessary when a user namespace applies to the workload.

Reply:
see comment at https://github.com/kubernetes/kubernetes/pull/118760/files#r1373287637